fga-eps-mds/2020.1-stay-safe-front-end
# Make some aspects clear to users

## Issue 
Resolves #82


## Description

Standardized the language to Portuguese, increased the DF map starting zoom, added the period of the data shown on the map, improved some texts for better understanding
## Screenshots (if appropriate):



## Checklist:



## Known problems (if appropriate):



## How to validate?

Go to Cadastro screen and Profile screen, check that Username is now Nome de usuário, also go to Rating and see the improved  "Se desejar, conte-nos o por quê:" text.
On ocurrences, the texts have also been improved, aligned also.
On Home, increased the amount of zoom of the map on DF, also added 2 variables to store the dates to inform the period of the data on the map.
